antora-playbook.yml

```diff
@@ -28,7 +28,44 @@ ui:
 
 output:
   clean: true
-  dir: ./build/site
-  
+  dir: ./site
 runtime:
   fetch: true
+
+# tag::attributes[]
+asciidoc:
+  attributes:
+    page-pagination: true
+    hide-uri-scheme: ""
+    kroki-server-url: 'http://kroki:8000'
+    kroki-fetch-diagram: ""
+    # Allow URI read to retrieve SVG on compilation
+    allow-uri-read: true
+    # use SVG unless otherwise specified
+    kroki-default-format: svg
+    # Make SVG inline so that we can get interactive (can be overridden by schema with "opts=none")
+    kroki-default-options: interactive
+    sectnums: "@"
+    xrefstyle: short@
+    listing-caption: Code
+    page-width-limit: none@
+    THIS_STANDARD: ASAM PLACEHOLDER
```

Comment:
What is this placeholder for?

Reply:
This placeholder was copied from the asam-antora-extension code - I've replaced it with the value for OpenMATERIAL.